auth/server/src/main/java/com/seagates3/controller/AccountLoginProfileController.java

```diff
@@ -182,10 +188,17 @@
           }
 
           if (requestBody.get("PasswordResetRequired") != null) {
+            isRequiredInputProvided = true;
             account.setPwdResetRequired(
                 requestBody.get("PasswordResetRequired").toUpperCase());
             LOGGER.info("Updating password reset required flag");
           }
+          if (!isRequiredInputProvided) {
+            LOGGER.error(
+                "Neither password not password-reset flag is provided");
```

Comment:
what if we add more arguments in future, these log messages will change, so better log as missing mandatory input arguments

Reply:
fixed